README.md

```diff
@@ -1 +1,3 @@
-# Hello-world
\ No newline at end of file
+# Hello-world
+
+hello world is the first project of microverse.

```

Comment:
- [ ] `required`
please make your readme more descriptive please follow the [requirements](https://github.com/microverseinc/readme-template/blob/master/README.md)